src/app/components/header.html

```diff
@@ -70,7 +70,7 @@
               </a>
             </li>
             <li>
-              <a href="https://fabric8.io/guide/getStarted/index.html">
+              <a href="https://fabric8.io/guide/getStarted/index.html" target="_blank">
```

Comment:
@ppalaga - it looks like you applied this to the header file that was left behind from a migration. Can you apply this change to index.html on line 93? You can delete header.html from src/app/components :-)